ads/vendors/gecko.md

```diff
@@ -0,0 +1,48 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Nit: 2021

Reply:
Hello Shihua,
Thank you very much, can you please check now? I've fixed all the issues you raised